cloudvolume/datasource/graphene/mesh/sharded.py

```diff
@@ -161,30 +165,33 @@ def get_meshes_via_manifest_byte_offsets(self, seg_id, bounding_box):
       mesh.segid = int(os.path.basename(f['path']).split(':')[0])
       dynamic_meshes.append(mesh)
 
-    fetches = []
-    segid_map = {}
-    for layer_id, filename, byte_start, size, segid in lists['initial']:
-      path = self.meta.join(layer_id, filename)
-      byte_end = byte_start + size
-      fetches.append({
-        'path': path,
-        'start': byte_start,
-        'end': byte_end,
-      })
-      segid_map[(path, byte_start, byte_end)] = segid
-
-    cloudpath = self.meta.join(self.meta.meta.cloudpath, self.meta.mesh_path, 'initial')
-    
-    files = CloudFiles(cloudpath, green=self.config.green).get(fetches)
-    initial_meshes = []
-    while files:
-      f = files.pop()
-      mesh = Mesh.from_draco(f['content'])
-      start, end = f['byte_range']
-      key = (f['path'], start, end)
-      mesh.segid = segid_map[key]
-      initial_meshes.append(mesh)    
-
+    if verify:
```

Comment:
Just as a style point, it's usually a not-so-good idea to put long blocks of code under an if/else because it forces the reader to remember which level of indentation they're in. For example, in this case it might be more readable to write:

```python
if not verify:
      segid_map = self.get_meshes_on_bypass(lists['initial'], allow_missing=True)
      initial_meshes = [ v for v in segid_map.values() ]
      return dynamic_meshes + initial_meshes

# verify code follows
```